README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
+
+Install/uninstall
+=================
+
+Run the command to install `sourcerer` tool on your machine:
+
+```
+curl -s https://sourcerer.io/install | sudo bash
+```
+
+Run the command to remove it from your machine:
+
+```
+curl -s https://sourcerer.io/uninstall | sudo bash
```

Comment:
Do you foresee we will go beyond the single sourcerer.jar? Otherwise this is a safe assumption I'd say. On the other hand a server side uninstall script could remove everything it installed, ever. This may be a reasonable approach, if the install script is more or less stable and not changed all the time.

Anyway, if you really prefer a local uninstall script, then it'd be a nice option to have something like:
sourcerer --uninstall
but not sure if the script can remove itself while running. What's your thinking?